zkevm-circuits/src/evm_circuit/step.rs

```diff
@@ -86,8 +86,8 @@ pub enum ExecutionState {
     SELFDESTRUCT,
     // Error cases
     ErrorInvalidOpcode,
-    ErrorStackOverflow,
-    ErrorStackUnderflow,
+    ErrorStack,
+    //ErrorStackUnderflow,
```

Comment:
//ErrorStackUnderflow, can be removed

Reply:
removed in [3c9c55d](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/873/commits/3c9c55da2df49e8f6a236241b8d42fcfe6f4f465) thanks!